.changeset/silly-clocks-tan.md

```diff
@@ -0,0 +1,63 @@
+---
+"@fluidframework/tree": major
+---
+
+Implement compatibility-based schema evolution API
+
+This change adjusts some top-level APIs for using SharedTree in a context that may necessitate changing a document's schema.
```

Comment:
There are two grammatically valid ways to parse this sentence with different meanings. One implies that the adjustment of the API may necessitate changing a document's schema: this is the kind of thing we commonly see in changesets but not what this is saying. so I think it should be rephrased.

```suggestion
This change adjusts some top-level APIs for using SharedTree to better accommodate applications that need to change their schema.
```

Reply:
By "there are two grammatically valid ways to parse this sentence with different meanings," are you trying to say that the sentence can be parsed in two ways, and each way has a different meaning? Or are you trying to say that the sentence, after being given different meanings, can be parsed in two ways?